prep-css-box-model/styles.css

```diff
@@ -9,10 +9,7 @@ button {
   /* 👇 YOUR CODE HERE 👇 */
   margin: 20px;
   padding: 10px;
-
-  p.double {
-    width: 3px;
-    border-style: double;
-    color: darkgray;
-  }
+  width: 3px;
+  border-style: double;
+  color: darkgray;
```

Comment:
To get these styles applied you'll want to use the properties i mentioned earlier

<img width="308" alt="image" src="https://user-images.githubusercontent.com/7432943/191870451-b3359d15-2a29-4437-b063-0c7365d4d821.png">